src/common/Extensions/IOperationContextExtensions.cs

```diff
@@ -3,22 +3,14 @@
 // Licensed under the MIT license.
 // --------------------------------------------------------------------------------------------
 
-using System.Net.Http;
+using k8s.Autorest;
 using Microsoft.BridgeToKubernetes.Common.Logging;
-using Microsoft.Rest;
-using Microsoft.Rest.Azure;
+using System.Net.Http;
 
 namespace Microsoft.BridgeToKubernetes.Common.Extensions
 {
     internal static class IOperationContextExtensions
     {
-        public static void SetRequestIds(this IOperationContext context, CloudException ex)

```

Comment:
Removing this overloaded method was intentional? Is it because it was not used anywhere ?

Reply:
@hsubramanianaks Yeah it wasn't in use and had "CloudException" as a in parameter so I think this was legacy code from DevSpaces.